pygount/analysis.py

```diff
@@ -521,6 +551,47 @@ def name_regexps_to_skip(self):
     def name_regexps_to_skip(self, regexps_or_pattern_text):
         self._name_regexp_to_skip = pygount.common.regexes_from(regexps_or_pattern_text, self.name_regexps_to_skip)
 
+    @staticmethod
+    def is_valid_git_link(git_link: str) -> bool:
+        # Regex to check valid  GIT Repository
+        # from https://stackoverflow.com/questions/2514859/regular-expression-for-git-repository
+        git_regex = re.compile(r"((git|ssh|http(s)?)|(git@[\w\.-]+))(:(\/\/)?)([\w\.@\:\/\-~]+)(\.git)(\/)?")
+
+        if git_regex.match(git_link):
+            return True
+        else:
+            return False
+
+    @staticmethod
+    def repo_arguments(git_link: str):
```

Comment:
Add result type.

Reply:
Updated in [86cbce5](https://github.com/roskakori/pygount/pull/111/commits/86cbce5e2f49caf80242b97c3aa5c0708b4abf03)